drivers/npmx/npmx_shell.c

```diff
@@ -3494,6 +3494,55 @@ static int cmd_timer_config_prescaler_get(const struct shell *shell, size_t argc
 
 	return timer_config_get(shell, TIMER_CONFIG_PARAM_PRESCALER);
 }
+static int timer_trigger_task(const struct shell *shell, npmx_timer_task_t task)
+{
+	npmx_timer_t *timer_instance = timer_instance_get(shell);
+	if (timer_instance == NULL) {
+		return 0;
+	}
+
+	npmx_error_t err_code = npmx_timer_task_trigger(timer_instance, task);
+
+	if (!check_error_code(shell, err_code)) {
+		print_set_error(shell, "timer task");
+		return 0;
+	}
+
+	print_success(shell, true, UNIT_TYPE_NONE);
+
+	return 0;
+}
+
+static int cmd_timer_start(const struct shell *shell, size_t argc, char **argv)
+{
+	ARG_UNUSED(argc);
+	ARG_UNUSED(argv);
+
+	return timer_trigger_task(shell, NPMX_TIMER_TASK_ENABLE);
+}
```

Comment:
```suggestion
static int cmd_timer_enable(const struct shell *shell, size_t argc, char **argv)
{
	ARG_UNUSED(argc);
	ARG_UNUSED(argv);

	return timer_trigger_task(shell, NPMX_TIMER_TASK_ENABLE);
}
```

As the enum is `ENABLE`, the function name should follow it. The same for other functions and command names.

Reply:
Changed. Also, moved strobe to config submenu